packages/lesswrong/components/comments/ModerationGuidelines/ForumModerationGuidelinesContent.js

```diff
@@ -0,0 +1,7 @@
+import { getSetting } from 'meteor/vulcan:core';
+import {frontpageGuidelinesLW, defaultGuidelinesLW} from './LWModerationGuidelinesContent'
+import {frontpageGuidelinesEA, defaultGuidelinesEA} from './EAModerationGuidelinesContent'
+
+export const frontpageGuidelines = getSetting('EAForum') ? frontpageGuidelinesEA : frontpageGuidelinesLW
```

Comment:
Can you explicitly set a default value here? I think it defaults to undefined here, which is falsy, but seems better to have explicit.

Reply:
Did you know it's actually impossible to set the default to a falsey value?

Reply:
Ah, yeah. I remember that. I guess in that case, let's leave it as is.